apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -115,6 +117,7 @@ def render(element_html, data):
             "editable": editable,
             "size": pl.get_integer_attrib(element, "size", SIZE_DEFAULT),
             "uuid": pl.get_uuid(),
+            "placeholder": placeholder,
```

Comment:
I don't think this needs to be included here?

Reply:
You're right, this was before I realised I could override the `shortinfo` variable